src/update_objects.c

```diff
@@ -489,18 +492,18 @@ s32 func_80072B48(s32 objectIndex, s32 arg1) {
     s32 phi_v1;
 
     phi_v1 = 0;
-    if (gObjectList[objectIndex].unk_0CB == 0) {
-        set_object_unk_0CB(objectIndex, 1);
+    if (gObjectList[objectIndex].isTimerRunning == 0) {
+        set_object_timer_running(objectIndex, 1);
```

Comment:
trigger_object_timer ?
or start_object_timer

Reply:
the name itself is fine. Just thought this might be better. You choose.

Reply:
for me trigger_object_timer or start_object_timer will don't get an bool to set and if we put at the 0 it will not start

Reply:
`toggle_timer_state` or `toggle_object_timer`?

Reply:
for me toggle it's automatically switch from true to false and vice-versa and don't have an extra arg to set it

Reply:
hmm set_object_timer_state ?

Reply:
Sure